### PR TITLE
Changed try! macros to ? operator

### PR DESCRIPTION
**Pull request by saschagrunert**
Since the stabilization of the ? operator (release 1.13.0)
the ? operator should be used to use idiomatic Rust.

### CODE REVIEW

**Comment by saschagrunert** [comment]
Is the build issue somehow related to my pull request? Since I think the master has the same problem.

File: src/bin/bench.rs [hunk 1]
```diff
@@ -71,12 +71,12 @@ Compilation can be customized with the `bench` profile in the manifest.
 ";
 
 pub fn execute(options: Options, config: &Config) -> CliResult<Option<()>> {
-    let root = try!(find_root_manifest_for_wd(options.flag_manifest_path, config.cwd()));
-    try!(config.configure(options.flag_verbose,
+    let root = find_root_manifest_for_wd(options.flag_manifest_path, config.cwd())?;
+    config.configure(options.flag_verbose,
                           options.flag_quiet,
```
Comment (alexcrichton) [inline src/bin/bench.rs]: This happened in quite a few places I think, but could the indentation be updated here?

Comment (saschagrunert) [inline src/bin/bench.rs]: For sure, I tried to get all of the wrong indentations.

**Comment by alexcrichton** [comment]
@bors: r+

Thanks!

cc @aturon, @nikomatsakis, fun changes with `?`!

**Comment by alexcrichton** [comment]
Oh we unfortunately haven't had a nightly for the past few nights, so a different date in `rustversion.txt` may be needed.

It also looks like tidy on travis may be turning up some problems?

**Comment by saschagrunert** [comment]
Alright, I guess these issues are now fixed too.

**Comment by alexcrichton** [comment]
Oh looks like 2016-11-04 isn't a nightly.

**Comment by saschagrunert** [comment]
I am sorry, `2016-11-05` should be a nightly right? If I check it with rustup it shows:

```
> rustup update
info: syncing channel updates for 'nightly-x86_64-unknown-linux-gnu'
info: checking for self-updates

  nightly-x86_64-unknown-linux-gnu unchanged - rustc 1.14.0-nightly (cae6ab1c4 2016-11-05)
```

**Comment by alexcrichton** [comment]
I think the last nightly to succeed was `nightly-2016-11-03`

**Comment by saschagrunert** [comment]
Thank you very much! 👌🏼

**Pull request merged**